src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -647,6 +648,7 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
   // Sends OnPermissionChange notification to mobile right after RAI response
   // and HMI level set-up
   GetPolicyHandler().OnAppRegisteredOnMobile(application->policy_app_id());
+  (*notify_upd_manager)();
```

Comment:
@LuxoftAKutsan 
Create typedef like StatusNotifier or some 'talking' name and remove this ugly thing. Add comment why it is here or I'm sure it will be removed in future.